docs/operator-guide/cd/deploy-rpm.md

```diff
@@ -0,0 +1,309 @@
+        │   └── 02_playbook.yml
+        └── cc
+            ├── 01_playbook.yml
+            ├── 02_playbook.yml
+            └── 03_playbook.yml
+```
+
+Below is a breakdown of the GitOps repository structure:
+
+- **nexus.repo**: Сontains the configuration for connecting to Nexus as a repository on the instances.
+- **inventory.ini**: Contains groups with instances, user credentials, and additional settings for connecting to the instances.
+- **pre-deploy.yml**: Contains general playbooks that contain a set of tasks that must be done before starting work, such as copying and connecting Nexus config, installing additional packages depending on the group.
+
+The rest of the files are example of deployment configurations. To spread these configuration files between Deployment Flows and Environments, files comply to the following hierarchy:
+
+```bash
+\<deployment-flow-name\>/\<environment-name\>/\<package-name\>/01_\<file\>.yml
+```
+
+For example, the **web/qa/nano/01_copy-file.yml** file relates to the **nano** application that belongs to the **qa** Environment, which in turn belongs to the **web** Deployment Flow. The file name must begin with its serial number (e.g., 01, 02, 03, ...), followed by an underscore, the rest of the file name, and the file extension.
+
+## AWX Approach
+
+The first approach to use RPM package is Default. This approach is considered more simple as it implies creating most of the resources via AWX user interface.
+
+### AWX Integration Flow
+
+Default approach involves the following steps:
+
+```mermaid
+graph LR;
+    A(Install AWX) --> B(Onboard Ansible Git repisitory) --> C(Configure AWX)--> D(Create application) --> E(Select deploy pipeline)
+```
+
+Here is a breakdown of the scheme above:
+
+1. **Onboard Ansible Git repository**: Create an Ansible Git repository that contains playbooks that will be used in deploy pipelines.
+2. **Install AWX**: Set Up AWX application. We recommend using [cluster add-ons](../add-ons-overview.md) to install it.
+3. **Configure AWX**: Once AWX tool is installed, open its UI to create AWX Project, Inventory, Templates, etc.
+4. **Create application**: [Add your application](../../user-guide/add-application.md) to KubeRocketCI as an RPM package.
+5. **Select deploy pipeline**: Choose a specific deploy pipeline when creating/editing an environment to deploy RPM packages.
+
+### AWX Integration Procedure
+
+Start off by ensuring that External Secrets Operator is [installed](../secrets-management/install-external-secrets-operator.md) and [configured](../secrets-management/external-secrets-operator-integration.md) properly.
+
+To set up RPM integration using AWX tool, follow the steps below. By completing these steps, you will be able to add, store, and deploy RPM applications:
+
+1. Onboard Ansible Git repository:
+
+  1. Create a repository where you will store you Ansible configuration files. Below is a structure of a GitOps repository designed for deploying RPM packages:
```

Comment:
Lets change "Below is a structure of a GitOps repository..." to "Below is a structure of an Ansible Git repository..."